libs/ui/src/lib/textarea/textarea.component.html

```diff
@@ -0,0 +1,19 @@
+<div>
+    <ng-container *ngIf="label !== ''">
+        <label [for]="name" class="block text-sm font-medium leading-6 text-gray-900">{{ label }}</label>
+    </ng-container>
+    <div class="mt-2">
```

Comment:
I'll suggest to keep the `mt-2` class tight coupled to the label value to keep the textarea component as clean as possible regarding external margins, paddings, etc.

```suggestion
    <div [ngClass]="{'mt-2': label}">
```